tasks/main.yml

```diff
@@ -1,7 +1,18 @@
 ---
+- name: Setting EPEL for armhfp to "{{ wtd_repo_epel_state }}"
```

Comment:
Normally you should add a "cleanup task" which is already there in the yum role in case somebody wants to remove the repository.
I would suggest adding a notify "Cleanup yum metadata" and the dependency to the yum role.
As the Ansible Galaxy thing is now finally fixed, you can depend on the yum role correctly.